fossology/uploads.py

```diff
@@ -216,19 +215,16 @@ def upload_licenses(self, upload, agent=None, containers=False):
         :rtype: list of licenses as JSON object
         :raises FossologyApiError: if the REST call failed
         """
+        params = {}
         if agent:
-            agent = agent.value
+            params["agent"] = agent.value
```

Comment:
Just a suggestion here, you can actually ask for multiple agents at once. Just pass them as a csv (nomos,monk,ojo).

Reply:
:+1: code is aligned now.